lib/addStyles.js

```diff
@@ -102,14 +102,13 @@ function addStylesToDom (styles, options) {
 		if(domStyle) {
 			domStyle.refs++;
 
-			for(var j = 0; j < domStyle.parts.length && item.parts.length; j++) {
```

Comment:
Nope I haven't, only used it locally for a few days and it seemed to work fine + `style-loader` tests. But I'm not using `webpack-hot-middleware` or the like